dialects/mysql/src/main/kotlin/app/cash/sqldelight/dialects/mysql/grammar/MySql.bnf

```diff
@@ -305,7 +305,7 @@ insert_stmt ::= [ {with_clause} ] ( INSERT | REPLACE ) [ 'LOW_PRIORITY' | 'DELAY
   pin = 7
 }
 
-assignment_value ::= VALUES LP {column_name} RP | { <<expr '-1'>> | DEFAULT}
+assignment_value ::= VALUES LP {column_name} RP |  <<expr '-1'>> | DEFAULT
```

Comment:
I still don't get it, why you have an -1 here and sometimes other values, like -6 🤔